### PR TITLE
added the check for response processed, and renamed success to send

### PR DESCRIPTION
**Pull request by Rutger798**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -168,8 +168,10 @@ pub struct Response {
...
 
 impl Response {
-    /// Verifies successful if there was a request
-    pub fn success(&self) -> bool {
+    /// Verifies if the date successful send
+    pub fn is_sent(&self) -> bool {
         self.response == "success"
     }
+    /// Verifies if the request is successful processed
+    pub fn is_processed(&self) -> bool { self.info.contains("processed: 1") }
```
Comment (ava57r) [inline src/lib.rs]: Maybe we should do an analysis of all the values, for example [code](https://github.com/andreevlex/zabbix-sender-cs/blame/master/ZabbixSender/SenderResponse.cs#L50), as did for us C#
add
- Processed
- Failed
- Total
- SecondsSpent

File: src/lib.rs [hunk 2]
```diff
@@ -168,8 +168,10 @@ pub struct Response {
 }
 
 impl Response {
-    /// Verifies successful if there was a request
-    pub fn success(&self) -> bool {
+    /// Verifies if the date successful send
+    pub fn is_sent(&self) -> bool {
```
Comment (ava57r) [inline src/lib.rs]: breaking change. Is it necessary to rename?

**Review by ava57r — CHANGES_REQUESTED** [review]

**Comment by Rutger798** [comment]
You have a valid point, so i have reverted the name change, and implemented the other status like you did in C#.

I'am bit new to rust and very new to github so just let me know if a should do something different.

**Review by ava57r — APPROVED** [review]

**Comment by ava57r** [comment]
Thanks

**Comment by Rutger798** [comment]
Thank you for merging!

**Pull request merged**